string NAME=rotate

ObjectDrone d
